roles/internal/deploy-user/tasks/main.yml

```diff
@@ -1,8 +1,10 @@
 - name: Ensure we have a deploy group
+  become: true
```

Comment:
I might be wrong, but I thought that all the tasks in this role should have become set already. Not sure where that happens though, and having them listed here explicitly certainly makes it more obvious.